docs/source/databases.rst

```diff
@@ -45,3 +46,57 @@ The previous command will download the following NCBI databases:
 
 After these files are downloaded, the ``taxdump.tar.gz`` tarball's files are extracted and the non-redundant protein database (``nr.gz``)
 is formatted as a diamond database (i.e. ``nr.dmnd``). This will significantly speed-up the ``diamond blastp`` searches.
+
+Genome Taxonomy Database (GTDB)
+###############################
+
+If you would like to incorporate the benefits of using the Genome Taxonomy Database.
+You may do this manually or using a few Autometa helper scripts. If you would like to use Autometa's
+scripts for this, you will first need to install Autometa (See :ref:`Installation`).
+
+You can either run the following script or manually download the respective databases.
+
+.. code-block:: bash
+
+    # First configure where you want to download the GTDB databases
+    autometa-config \
+        --section databases --option gtdb \
+        --value <path/to/your/gtdb/database/directory>
+
+    # To use a specific GTDB release
+    autometa-config \
+        --section gtdb --option release \
+        --value latest
+        # Or --value r207 or --value r202, etc.
+
+    # Download and format the configured GTDB databases release
+    autometa-update-databases --update-gtdb
+
+
+.. note::
+
+    You can check the default config paths using ``autometa-config --print``.
+
+    See ``autometa-update-databases -h`` and ``autometa-config -h`` for full list of options.
+
+The previous command will download the following GTDB databases and format the `gtdb_proteins_aa_reps.tar.gz` to be used by Autometa:
+
+- Amino acid sequences of representative genome
+    - `gtdb_proteins_aa_reps.tar.gz <https://data.gtdb.ecogenomic.org/releases/latest/genomic_files_reps/gtdb_proteins_aa_reps.tar.gz>`_
+- gtdb-taxdump.tar.gz from `shenwei356/gtdb-taxdump <https://github.com/shenwei356/gtdb-taxdump/releases>`_
+    - `gtdb-taxdump.tar.gz <https://github.com/shenwei356/gtdb-taxdump/releases/latest/download/gtdb-taxdump.tar.gz>`_
+
+
+Once unzipped `gtdb-taxdump.tar.gz` will have the taxdump files of all the respective GTDB releases. Make sure that the release you use is in line with the `gtdb_proteins_aa_reps.tar.gz` release version. It's better to always use the latest version. 
```

Comment:
This section is not clear and you should break this in to multiple lines.

Where does `gtdb.faa` come in? When is `gtdb.dmnd` generated, if at all?